src/unix/apple/macosx/app.m

```diff
@@ -1988,6 +1988,10 @@ MTY_Size MTY_WindowGetScreenSize(MTY_App *app, MTY_Window window)
 		return (MTY_Size) {0};
 
 	NSSize size = ctx->nsw.screen.frame.size;
+	if (@available(macOS 12.0, *)) {
```

Comment:
According to the [documentation](https://developer.apple.com/documentation/appkit/nsview/safeareainsets?language=objc), this is available on macOS 11.0